{{cookiecutter.pipeline_slug}}/main.nf

```diff
@@ -26,7 +26,7 @@ def helpMessage() {
     Mandatory arguments:
       --reads                       Path to input data (must be surrounded with quotes)
       --genome                      Name of iGenomes reference
-      -profile                      Hardware config to use. docker / aws
+      -profile                      Hardware config to use. docker / awsbatch
```

Comment:
This message is a bit misleading. Should probably be "Configuration profile to use.", and list _all_ of the available options.